### PR TITLE
Update profile titles and descriptions

### PR DESCRIPTION
**Pull request by shawndwells**
mark draft profiles with [DRAFT] in profile title, comment them out from build system

### CODE REVIEW

**Comment by shawndwells** [comment]
@mpreisler properly rebased... i think

**Comment by redhatrises** [comment]
Rather than comment out in guide.xslt. I wonder if it would make more sense to have the build system detect if a profile is in draft form and ignore it on build.

File: rhel7/guide.xslt [hunk 1]
```diff
@@ -21,27 +21,29 @@
...
+      <!-- <xsl:apply-templates select="document('profiles/c2s-docker.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-rhvh-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-ansible-tower-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-http-disa.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-ipa-server-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/stig-satellite-upstream.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_minimal.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_intermediary.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_enhanced.xml')" /> -->
+      <!-- <xsl:apply-templates select="document('profiles/anssi_nt28_high.xml')" /> -->
```
Comment (mpreisler) [inline rhel7/guide.xslt]: this won't do anything in the new build system. I am surprised this file is even still in there

**Comment by shawndwells** [comment]
Created https://github.com/OpenSCAP/scap-security-guide/issues/2719 to track how to handle draft profiles. Reverted to old guide.xslt to not hold up this pr.

**Comment by shawndwells** [comment]
Not understanding the jenkins build errors. So many error findings about missing files... @mpreisler can you help me decipher what the real issue is?

**Comment by shawndwells** [comment]
Found this in the local build:
`````
Built target generate-all-roles-rhel7-sh
[ 40%] Built target generate-all-roles-rhel7-yml
Profile 'stig-rhel7-disa' not found.
make[3]: *** [tables/table-rhel7-stig-testinfo.html] Error 10
make[2]: *** [rhel7/CMakeFiles/generate-rhel7-table-stig.dir/all] Error 2
make[2]: *** Waiting for unfinished jobs....
`````

Backed out the profile renaming. Nope'ing myself out of dealing with CMake hell for now.

**Comment by redhatrises** [comment]
@shawndwells `documentation_complete` probably should be changed to `false` in the profiles.

**Comment by redhatrises** [comment]
@shawndwells can you fix the conflict in this PR?

**Comment by shawndwells** [comment]
@redhatrises conflicts resolved

**Comment by mpreisler** [comment]
@shawndwells please rebase this to remove merge commit and the reverts. Let's not even have the changes you reverted in the PR and in git history.

**Comment by shawndwells** [comment]
@mpreisler using the GitHub WebUI to resolve the conflicts. What your asking doesn't appear possible.

**Comment by yuumasato** [comment]
@shawndwells I'm afraid you'll have to use git through CLI, to rebase into master. And squashing the commits may be easier then dropping changes, and reverts.

**Comment by shawndwells** [comment]
don't have time

**Comment by jamescassell** [comment]
I'll do the rebase.

**Comment by jamescassell** [comment]
@shawndwells if you use `git revert <bad commit>` when you revert a change, it makes it a lot easier to rebase later.  For typos, you can do `git commit --fixup ':/^title of commit with typo'`, then you can later `git rebase --autosquash -i` and it will automatically squash it into the relevant commit.

See #2776 for the results of my rebase.

**Pull request closed without merging**